posts/2021-12-15-read-AWS.ipynb

```diff
@@ -0,0 +1,2712 @@
+{
```

Comment:
<p>"is stored <s>as</s> on Open"</p><p> </p><p>"<s>To</s> The name of"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>